developer_docs/ansible_dev_setup.asciidoc

```diff
@@ -65,6 +65,7 @@ Enable feature
 # all configuration options
 ---
 :enabled: true
+:working_dir: '/path/to/foreman_ansible'
```

Comment:
In the [(https://github.com/theforeman/smart_proxy_ansible/blob/master/settings.d/ansible.yml.example](ansible.yml.example) the example dir is `'~/.foreman-ansible'`. I don't think it should point to the foreman_ansible plugin dir.

Reply:
Why not? Is it used somewhere else? 

During my installation of foreman_ansible, I encountered an issue where running the application without this specific line resulted in failure. Therefore, I recommend updating the line not only in the foreman_ansible repository but also in the smart_proxy_ansible repository.

Reply:
> Why not? Is it used somewhere else?
Because it already contains files for the plugin, it shouldn't be used as a working dir.

> I encountered an issue where running the application without this specific line resulted in failure. 

`working_dir` must be set to foreman_ansible plugin or it must be set to _other_ existing ideally empty folder?

Reply:
I see now that if I don't specify the `working_dir`, it runs without issues. 
The problem is when you give it the wrong path. 

Maybe we should update this line in the `ansible.yml.example` file located in the smart_proxy_ansible repo?

Reply:
> Maybe we should update this line in the ansible.yml.example file located in the smart_proxy_ansible repo?

That file is correct, here we just have to point to another existing dir, different than the plugin dir.

Reply:
Sorry, I'm confused now. What is your recommendation? 
I submitted this pull request because I noticed someone encountering an unclear error due to an incorrect configuration while following the guide.
However, it might be enough to add a comment to clarify this, instead of changing the configuration file.

Reply:
Ok, I will try to explain.

**TLDR;**
Do not touch the installation directory of your plugin.

Your example has `working_dir` with the path to the `foreman_ansible` directory, where are folders and files for the `foreman_ansible` plugin, for example, `lib`, `app`, `Gemfile`, and so on. Simply it's the installation directory of the plugin.

As my understanding is, `working_dir` is a directory where Ansible is putting its data when it's running a playbook/role and does all other different stuff.

My point is, you **don't** want to put some temporary working files in the installation directory (`/path/to/foreman_ansible`) of your plugin, right? What will happen if Ansible rewrites/deletes something? Answer: you will break something.

What you want to do, is to have some empty directory where Ansible can do whatever it wants to, like
`/temp/ansible_work_dir/`.

Reply:
Thanks! That's much more clear more. 
Closing this PR.